website/src/styles/custom.css

```
@@ -0,0 +1,41 @@
+/* Noto Sans SC and M PLUS 1 Code provide CJK fallback coverage for the configured UI and code stacks. */
+/* The light CTA foreground uses resin-900 for contrast; --slate-on-dark preserves legibility on dark resin surfaces. */
+@import url("https://fonts.googleapis.com/css2?family=Manrope:wght@400;500;600;700;800&family=JetBrains+Mono:wght@400;500;600&family=Noto+Sans+JP:wght@400;700&family=Noto+Sans+SC:wght@400;700&family=M+PLUS+1+Code:wght@400;500;600&display=swap");
```

Stylelint (17.14.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Manrope:wght@400;500;600;700;800&family=JetBrains+Mono:wght@400;500;600&family=Noto+Sans+JP:wght@400;700&family=Noto+Sans+SC:wght@400;700&family=M+PLUS+1+Code:wght@400;500;600&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Manrope:wght@400;500;600;700;800&family=JetBrains+Mono:wght@400;500;600&family=Noto+Sans+JP:wght@400;700&family=Noto+Sans+SC:wght@400;700&family=M+PLUS+1+Code:wght@400;500;600&display=swap"" (import-notation)

(import-notation)